sample/jvm/src/main/java/co/yml/ychat/jvm/services/YChatService.java

```diff
@@ -65,6 +67,14 @@ public AIModel getModel(String id) throws Exception {
         return future.get();
     }
 
+    public String getAudioTranscription(MultipartFile multipartFile) throws Exception {
+        final CompletableFuture<String> future = new CompletableFuture<>();
+        String filename = Optional.ofNullable(multipartFile.getOriginalFilename()).orElse("");
```

Comment:
If there is no filename this will likely not work right? Shouldn't we raise an exception here?

Reply:
Good catch! Adjusted!